recipes-wpe/wpeframework/wpeframework-tools-native_git.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Host/Native tooling for the Web Platform for Embedded Framework"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1fe8768cbb5fd322f7d50656133549de"
+
+PR = "r0"
+PV = "3.0+git${SRCPV}"
+S = "${WORKDIR}/git"
+
```

Comment:
You can probable include `wpeframework.inc` for a lot of these and just provide the native tool specific values.

Reply:
Yeah but it sets the CMAKE_SYSROOT, I think it could mess up installation, better add some lines extra then to be sorry.